### ISSUE TITLE
oracledb_session.rb missing closing " from line 83

### ISSUE THREAD

**Issue by chris-gooch** [inspec/inspec#4847]
# Describe the problem
Line 83 of file "inspec/lib/inspec/resources/oracledb_session.rb" is missing the closing " and therefore not working. It is also not throwing and errors or exceptions.

## Possible Solution
Change line to include closing double-quote:

```ruby
%{su - #{@su_user} -c "env ORACLE_SID=#{@service} #{bin} / as #{@db_role}#{sql_postfix}"}
```

**Comment by miah** [comment]
Which version of InSpec are you using, it looks like you'll want to update. This issue was corrected with #4654

**Comment by miah** [comment]
Actually, It looks like this bug may still exist! I'll PR momentarily.
